inc/modules/export/htmlbook/class-htmlbook.php

```diff
@@ -723,10 +735,20 @@ protected function title( $book, $book_contents, $metadata ) {
 		if ( ! $content ) {
 			$content .= sprintf( '<h1 class="title">%s</h1>', get_bloginfo( 'name' ) );
 			$content .= sprintf( '<p class="subtitle">%s</p>', ( isset( $metadata['pb_subtitle'] ) ) ? $metadata['pb_subtitle'] : '' );
-			$content .= sprintf( '<p class="author">%s</p>', ( isset( $metadata['pb_author'] ) ) ? $metadata['pb_author'] : '' );
-			$content .= sprintf( '<p class="contributing-authors">%s</p>', ( isset( $metadata['pb_contributing_authors'] ) ) ? $metadata['pb_contributing_authors'] : '' );
+			if ( isset( $metadata['pb_authors'] ) ) {
+				$authors = oxford_comma_explode( $metadata['pb_authors'] );
+				foreach ( $authors as $author ) {
+					$content .= sprintf( '<p class="author">%s</p>', $author );
+				}
+			}
+			if ( isset( $metadata['pb_contributors'] ) ) {
```

Comment:
While we have to include this for historical reasons in EPUB and XHTML 1.1 I would like to remove it here. When discussing new book theme conventions with Hugh and Zoe, we talked about the potential for including editors, etc. on the cover page; I don't want to include anything other than authors _by default_ until we've spec'ed that feature properly.